src/Pyz/Zed/Sales/Presentation/Detail/boxes/customer.twig

```diff
@@ -0,0 +1,29 @@
+{% embed '@Gui/Partials/widget.twig' with { widget_title: 'Customer' | trans } %}
+    {% block widget_content %}
+        <div class="row">
+            <div class="col-xs-12 col-md-6">
+                <dl>
+                    <dt>{{ 'Reference' | trans }}</dt>
+                    <dd><a href="{{ url('/customer/view', {'id-customer': order.fkCustomer}) }}" target="_blank">{{ order.customerReference }}</a></dd>
+                    <dt>{{ 'Name' | trans }}</dt>
+                    <dd>{{ order.firstName ~ ' ' ~ order.lastName }}</dd>
+                    <dt>{{ 'Email' | trans }}</dt>
+                    <dd>{{ order.email }}</dd>
+                    <dt>{{ 'Previous orders count' | trans }}</dt>
+                    <dd>{{ order.totalOrderCount }}</dd>
```

Comment:
Shouldn't it be the same as at the core?
```suggestion
                    <dd>{{ order.totalOrderCount | executeFilterIfExists('formatInt', app.locale) }}</dd>
```

Reply:
This a part December release, not this one.